modules/candidate_list/php/candidate_list.class.inc

```diff
@@ -260,7 +260,7 @@ class Candidate_List extends \NDB_Menu_Filter
                                        );
 
         $participant_status_options = array(null => 'All')
-            +  \NDB_Form_candidate_parameters::getParticipantStatusOptions();
+            +  CP\candidate_parameters::getParticipantStatusOptions();
```

Comment:
I think this was moved to the Candidate class in the major branch. Maybe it would be better to just send a PR there and avoid conflicts?

Reply:
@driusan Do you mean to include the moving of getParticipantStatus in the Candidate class in this PR instead of having it in the major branch?

Reply:
No, I mean send this PR to the major branch, where the move is already done, rather than having conflicts that we're going to have to fix when we merge minor up into major.